gneiss/plot/_dendrogram.py

```diff
@@ -0,0 +1,235 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""Drawing trees.
+
+Draws horizontal trees where the vertical spacing between taxa is
```

Comment:
is this code about the construction of the tree representation or a layouting algorithm?
More suitable wording might be: Draws trees horizontally, i.e. root left, tips right, ....